src/main/java/uk/gov/ons/ctp/response/casesvc/CaseSvcApplication.java

```diff
@@ -247,4 +252,19 @@ public DistributedLockManager reportDistributedLockManager(RedissonClient rediss
   public Clock clock() {
     return Clock.systemDefaultZone();
   }
+
+  @Bean
+  public CacheManager cacheManager() {
+    return new ConcurrentMapCacheManager(COLLEX_CACHE_NAME);
+  }
+
+  @CacheEvict(
```

Comment:
Can this comment  be firmer than 'we imagine that it won't cause issues'. 
If we're not 100% confident then the code should be more secure?